gallery-so/gallery
# add bottom sheet context

### Summary of Changes
Add a bottom sheet context + actions to achieve 2 things:
- make it simple to add a bottom sheet anywhere with less boilerplate
- more tightly contain business logic within bottom sheets so they're not executed until the sheet is active

### Demo or Before/After Pics

see loom on slack


### Edge Cases

List any common edge cases that you have considered and tested.

### Testing Steps
- Go to profile > settings and tap the feedback button. this uses the new stuff so it should open as usual.
- Also test other unchanged bottom sheets to confirm that backward compatibility works and nothing regressed

### Checklist

Please make sure to review and check all of the following:
